scripts/package.ts

```diff
@@ -106,6 +107,68 @@ function getVersionSuffix(feature: string, debug: boolean): string {
     return `${debugSuffix}${featureSuffix}${commitSuffix}`
 }
 
+/**
+ * @returns true if curl is available
+ */
+function isCurlAvailable(): boolean {
+    try {
+        child_process.execFileSync('curl', ['--version']);
+        return true;
+    } catch {
+        return false;
+    }
+}
+
+/**
+ * Small utility to download files.
+ */
+function downloadFiles(urls: string[], outputDir: string, outputFile: string): void {
+    if (platform() !== 'linux') {
+        return;
+    }
+
+    if (!isCurlAvailable()) {
+        return;
+    }
+
+    // Create output directory if it doesn't exist
+    if (!nodefs.existsSync(outputDir)) {
+        nodefs.mkdirSync(outputDir, { recursive: true });
+    }
+
+    urls.forEach(url => {
+        const filePath = path.join(outputDir, outputFile || '');
+
+        try {
+            child_process.execFileSync('curl', ['-o', filePath, url]);
+        } catch {}
```

Comment:
:stop_sign: **Security Vulnerability**: Potential remote code execution risk. The `downloadFiles` function downloads and executes files from a remote source without proper validation[^1]. This could lead to the execution of malicious code if the source is compromised.
```suggestion
urls.forEach(url => {
    const filePath = path.join(outputDir, outputFile || '');
    
    // Validate URL
    if (!url.startsWith('')) {
        console.error(`Invalid URL: ${url}`);
        return;
    }

    try {
        // Use a more secure method to download files
        const response = await fetch(url);
        if (!response.ok) {
            throw new Error(`HTTP error! status: ${response.status}`);
        }
        const content = await response.text();
        await fs.promises.writeFile(filePath, content, 'utf-8');
    } catch (error) {
        console.error(`Failed to download file: ${error}`);
    }
})
```

[^1]: CWE-494: Download of Code Without Integrity Check - https://cwe.mitre.org/data/definitions/494.html